src/display/annotation_layer.js

```diff
@@ -345,21 +345,21 @@ class AnnotationElement {
         const hidden = event.detail.display % 2 === 1;
         this.container.style.visibility = hidden ? "hidden" : "visible";
         this.annotationStorage.setValue(this.data.id, {
-          hidden,
-          print: event.detail.display === 0 || event.detail.display === 3,
+          noView: hidden,
+          noPrint: event.detail.display === 1 || event.detail.display === 2,
```

Comment:
Obviously a pre-existing issue, but can you please add a comment explaining the "magical" values that we compare against in this function?